iFixit/react-commerce
# Troubleshooting TOC: Polish

This pull addresses many minor issues noticed when working on #1923

Some notes for CR:

@danielbeardsley brought up [a convo here](https://github.com/iFixit/react-commerce/pull/1923#discussion_r1301954469), which is still worth discussing.

CR with ignoring whitespace for an easier time.

QA
---

* Ensure TOC looks like the figma
* [Ensure TOC behaves](https://github.com/iFixit/react-commerce/pull/1923#issuecomment-1688960687)
* [Ensure mobile TOC scrolling on iOS works nicely](https://github.com/iFixit/react-commerce/pull/1923#issuecomment-1683192508)
* Point out anything of concern

Note: The causes list will be hidden on desktop due to work from @andyg0808 

Closes: https://github.com/iFixit/ifixit/issues/49284
